style.css

```
@@ -0,0 +1,172 @@
+html {
+  font-family: "Pretendard", sans-serif;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.layout {
+  max-width: 1120px;
+  min-width: 1120px;
+  margin: 0 auto;
+}
+
+.button {
+  background-color: #3692ff;
+  color: #ffffff;
+  font-weight: 600;
+  text-decoration: none;
+  display: inline-block;
+}
+
+.button:hover {
+  background-color: #1967d6;
+}
+
+.button:active {
+  background-color: #1251aa;
+}
+
+.header-container {
+  max-width: 1520px;
+  height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.logo-wrapper {
+  margin: 0 0 0 200px;
```

💊 제안
한 방향만 margin을 주시는 거라면 아래처럼 하시는 것이 더 짧고 가독성도 좋다고 생각합니다~

```suggestion
  margin-left: 200px;
```